app/assets/javascripts/discourse/components/text-overflow.js.es6

```diff
@@ -7,6 +7,7 @@ export default Component.extend({
       const $this = $(this.element);
 
       if ($this) {
+        $this.find("br").replaceWith(" ");
```

Comment:
Are `br` and `hr` the only tags that are breaking the `ellipsis()` method?

Reply:
I couldn't find any other element that could break the `ellipsis()` method.